components/x-teaser-timeline/src/TeaserTimeline.scss

```diff
@@ -0,0 +1,61 @@
+@import 'o-colors/main';
+@import 'o-grid/main';
+@import 'o-typography/main';
+
+:global {
```

Comment:
As per #221 it might be worth a comment stating that this could cause duplicate output.

(to avoid this perhaps the docs could instruct implementers to include the save button styles manually?)